test/classes/initializers/phase1/grandparentAccess.chpl

```diff
@@ -10,21 +10,20 @@ class B : A {
   var b = false;
 
   proc init(bVal, aVal) {
-    b = bVal;
     super.init(aVal);
+    b = bVal;
   }
 }
 
 class C : B {
   var c = 4;
 
   proc init(cVal, bVal, aVal) {
-    c = a;
+    var x = a;
```

Comment:
Let's leave a comment on this access in the file itself, so some future reader doesn't try to remove it due to thinking it's unnecessary to the purpose of the test